### PR TITLE
Task/ev testing

### PR DESCRIPTION
**Pull request by SteveMaier-IRT**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
*Link your PR to an issue*

Fixes #___

### Description
*Please describe your pull request.*

### Testing
*Describe the best way to test or validate your PR.*

### CODE REVIEW

**Comment by SteveMaier-IRT** [comment]
John and I discussed merging this before the review.

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Text.Json;
 using CommunityToolkit.Maui.Storage;
 using CommunityToolkit.Mvvm.Input;
+using ElectionGuard.Ballot;
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs:5]: unneeded using

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs [hunk 2]
```diff
@@ -421,15 +421,31 @@ private void CeremonyPollingTimer_Tick(object? sender, EventArgs e)
...
+                    });
+                    await UpdateTallyData();
+#if DEBUG
+                    ErrorMessage = Tally.State.ToString();
+#endif
+                    if (await _tallyRunner.Run(localTally))
                     {
                         ErrorMessage = string.Empty;
+#if DEBUG
+                        ErrorMessage = $"{Tally.State.ToString()} ran";
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs:441]: this shouuld use localTally

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs [hunk 3]
```diff
@@ -421,15 +421,31 @@ private void CeremonyPollingTimer_Tick(object? sender, EventArgs e)
...
+#endif
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs:442]: don't need this

**Review by john-s-morgan — COMMENTED** [review]
nits

**Pull request merged**